scripts/PayloadDeployment.s.sol

```diff
@@ -26,7 +31,7 @@ contract DeployAvalanche is Script {
 
     vm.startBroadcast();
     RewardsController rewardsControllerImpl = new RewardsController(emissionManager);
-    rewardsControllerImpl.initialize(emissionManager);
+    rewardsControllerImpl.initialize(address(0));
```

Comment:
update initialize to use `address(0)`, as requested by @miguelmtzinf (in reality doesn't matter)